src/main/java/bio/terra/app/controller/SnapshotsApiController.java

```diff
@@ -171,9 +172,16 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
     }
 
     @Override
-    public ResponseEntity<SnapshotModel> retrieveSnapshot(@PathVariable("id") String id) {
+    public ResponseEntity<SnapshotModel> retrieveSnapshot(
+        @PathVariable("id") String id,
+        @Valid @RequestParam(
+            value = "include",
+            required = false,
+            defaultValue = "SOURCES,TABLES,RELATIONSHIPS,PROFILE,DATA_PROJECT"
```

Comment:
I'm wondering what the best practice for overrides and annotations like this is. Recently there was a PR that fixed an issue where the `defaultValue` annotation changed in the openapi-generated interface but was never changed in the controller override. The compiler/IDE didn't flag this as an error. If the code works correctly without copying down the annotations, it seems safer to not copy them down.

Reply:
Yeah, I was sad that I had to put this in there but it looked like it wasn't getting the default values otherwise

Reply:
I looked in the docs and you're right; it's required. I wish there was a way to make the compiler raise an error if the annotations disagreed. Having us keep them in sync manually isn't good.

After googling a bit more I found a tool that helps detect API breaking changes to swagger, maybe that would be a tool to help us detect changes that require updating annotations. https://github.com/civisanalytics/swagger-diff